src/assets/scripts/client/game/GameController.js

```diff
@@ -608,6 +610,23 @@ class GameController {
 
         this.theme = THEME[themeName];
     };
+
+    /**
+     * Retrieves the current version of the simulator from the `window` global object.
+     *
+     * @for GameController
+     * @method _getVersion
+     * @return {string}
+     */
+    _getVersion() {
+        const version = window.GLOBAL.VERSION;
```

Comment:
is there ever a case where this wouldn't actually be set?  This is set via build tool, so I don't think it would *never* be defined

Reply:
Probably not, I was just being needlessly overcautious

Also sorry for requesting review, just force of habit (it's pretty much done anyway)

Reply:
reviews are required, so you did good there 😉